spek-ide-plugin/intellij-idea/src/IJ182/kotlin/org/spekframework/intellij/SpekJvmRunConfiguration.kt

```diff
@@ -0,0 +1,47 @@
+package org.spekframework.intellij
+
+import com.intellij.execution.Executor
+import com.intellij.execution.configurations.*
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.module.ModuleManager
+import com.intellij.openapi.options.SettingsEditor
+import org.jdom.Element
+import org.spekframework.intellij.support.SpekJvmCommonRunConfigurationParameters
+import java.util.*
+
+class SpekJvmRunConfiguration(name: String, module: JavaRunConfigurationModule, factory: ConfigurationFactory)
+    : ModuleBasedConfiguration<JavaRunConfigurationModule>(name, module, factory)
```

Comment:
this is the breaking change: `ModuleBasedConfiguration<JavaRunConfigurationModule>` from `ModuleBasedConfiguration`.. fun timess.

Reply:
Not!